config/lookoutingesterv2/config.yaml

```diff
@@ -0,0 +1,28 @@
+postgres:
+  maxOpenConns: 100
```

Comment:
we can probably reduce this- the new iingester  will only have a single thread ingesting into the database at any one time so should (in theory) only need a single connection.  We can have a pool to give us flexibility later if we need it, but I think we can reduce it to something like a fixed pool of 5 connections.

(I realise you probably just copied this from the original ingester config!)